spec/app/models/mdm/host_spec.rb

```diff
@@ -1,4 +1,6 @@
 RSpec.describe Mdm::Host, type: :model do
+  include_context 'Rex::Text'
```

Comment:
We don't need to hold up the PR b/c of this, but it's a good general note, @limhoff-r7.

Are we good to go here otherwise? If so, let's get this landed ASAP.